api/custom/datastores.go

```diff
@@ -26,25 +26,25 @@ type DatastoreMigration struct {
 
 func GetDatastores(r *http.Request, rctx rcontext.RequestContext, user _apimeta.UserInfo) interface{} {
 	response := make(map[string]interface{})
-	for _, ds := range config.UniqueDatastores() {
-		uri, err := datastores.GetUri(ds)
+	for _, store := range config.UniqueDatastores() {
+		uri, err := datastores.GetUri(store)
 		if err != nil {
 			sentry.CaptureException(err)
 			rctx.Log.Error("Error getting datastore URI: ", err)
 			return _responses.InternalServerError(errors.New("unexpected error getting datastore information"))
 		}
-		dsMap := make(map[string]interface{})
-		dsMap["type"] = ds.Type
-		dsMap["uri"] = uri
-		response[ds.Id] = dsMap
+		dataStoreMap := make(map[string]interface{})
+		dataStoreMap["type"] = store.Type
+		dataStoreMap["uri"] = uri
+		response[store.Id] = dataStoreMap
 	}
 
 	return &_responses.DoNotCacheResponse{Payload: response}
 }
 
 func MigrateBetweenDatastores(r *http.Request, rctx rcontext.RequestContext, user _apimeta.UserInfo) interface{} {
 	beforeTsStr := r.URL.Query().Get("before_ts")
-	beforeTs := util.NowMillis()
+	beforeTs := time.Now().UnixNano() / int64(time.Millisecond)
```

Comment:
[Relevant XKCD](https://www.explainxkcd.com/wiki/index.php/2867:_DateTime) Though, I think it would make sense to copy the reasoning in here. This isn't the latest version though, so I would argue the benefits of `time.NowMillis` that a reimplementation (or an alias) of `time.Now().UnixMilli()` is unnecessary.

Reply:
Normally I would agree, however for the operations we're doing here in MMR we really do not want to use `time` or overload `time` as a package name. `util.NowMillis()` is semantically useful as-is, and clarifies exactly what precision we need.